sycl/include/CL/sycl/INTEL/fpga_lsu.hpp

```diff
@@ -92,6 +97,14 @@ template <class... _mem_access_params> class lsu final {
 
   static_assert(_cache_val >= 0, "cache size parameter must be non-negative");
 
+  template <access::address_space _space> static void check_space() {
+    static_assert(_space == access::address_space::global_space ||
+                      _space == access::address_space::global_device_space ||
+                      _space == access::address_space::global_host_space,
```

Comment:
For double check: does LSU support ptr in global host address space?

Reply:
Yes it does